engines/hypno/hypno.cpp

```diff
@@ -235,6 +235,31 @@ void HypnoEngine::drawImage(Graphics::Surface &surf, int x, int y, bool transpar
 		_compositeSurface->blitFrom(surf, Common::Point(x, y));
 }
 
+Common::File *HypnoEngine::fixSmackerHeader(Common::File *file) {
```

Comment:
Maybe this method should also just deal with `Common::ReadStream`.

Reply:
When this function is used, a pointer to an open file is always available, so it is handy to use here.